interfaces/portal/src/app/components/query-table/query-table.component.ts

```diff
@@ -325,9 +325,9 @@ export class QueryTableComponent<TData extends { id: PropertyKey }, TContext> {
     return !!tableFilterMetadata?.value;
   }
 
-  getColumnMatchMode(column: QueryTableColumn<TData>) {
+  getColumnMatchMode(column: QueryTableColumn<TData>): FilterMatchMode {
     if (column.filterMatchMode) {
-      return column.filterMatchMode as string;
+      return column.filterMatchMode;
     }
```

Comment:
Does this situation ever occur? When I search for filterMatchMode in the code I just get three occurrences, but maybe I am missing something.

If this does not occur we can get rid of the if and rename `getColumnMatchMode` to `getDefaultColumnMatchMode`

Reply:
Good call, I got rid of couple other unused attributes too. I guess they became redundant with some new features/refactoring.

I won't rename the function to `getDefault` thought, because `getColumnX` remains in line with all of the other functions.